config.go

```diff
@@ -145,6 +145,8 @@ func (c *Config) Default() error {
 	switch {
 	case util.Exists("main.go"):
 		golang(c)
+	case util.Exists("pom.xml") || util.Exists("build.gradle"):
+		java(c);
```

Comment:
You could remove this duplication of checking for `pom.xml` or `build.gradle` by creating 2 functions, `javaGradle` and `javaMaven`.

Reply:
Good suggestion. I'll push the change shortly.